Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -2403,7 +2403,7 @@ private void SetFinalColumnsToSelect(InternalItemsQuery query, List<string> colu
                 }
 
                 // genres, tags, studios, person, year?
-                builder.Append("+ (Select count(1) * 10 from ItemValues where ItemId=Guid and CleanValue in (select CleanValue from itemvalues where ItemId=@SimilarItemId))");
+                builder.Append("+ (Select count(1) * 10 from ItemValues where ItemId=Guid and CleanValue in (select CleanValue from ItemValues where ItemId=@SimilarItemId))");
```

Comment:
Unnecessary change, no? I'm pretty sure it's case insensitive

Reply:
Me too, but we did it case sensitive for all other tables except this one, so for consitencie's sake I changed it. I cna revert that if required.